.github/workflows/e2e-tests.yml

```diff
@@ -25,9 +25,9 @@ jobs:
       - uses: actions/checkout@v2
       - id: create-matrix
         run: |
-          MATRIX='[{"group":"pre-connection"},{"group":"connection"},{"group":"post-connection"}]'
+          MATRIX='[{"group":"pre-connection","tests":"pre-connection"},{"group":"connection","tests":"/connection"},{"group":"post-connection","tests":"post-connection"}]'
```

Comment:
Im just starting the review, so bear with me. 
`"tests":"/connection"` seems inconsistent with other parts of the matrix. Is this intentional?

Reply:
That's a good catch. It's probably a copy/paste issue. Fixed with [85e0be4](https://github.com/Automattic/jetpack/pull/21848/commits/85e0be4ecbe57a4c96e6f8d1fca77c3b54d308af)